packages/react-ai/src/ai-conversation-styles.css

```diff
@@ -125,4 +126,59 @@
   align-items: center;
   gap: 4px;
   overflow: hidden;
+  cursor: pointer;
+}
+
+.ai-input__container {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  gap: 8px;
+  height: 40px;
+}
+
+.ai-input__input {
+  align-self: stretch;
+  color: var(--colors-font-primary, #0d1a26);
+  border-radius: 4px;
+  border: 1px solid var(--components-fieldcontrol-borderColor, #aeb3b7);
+  background: #fff;
+  width: 800px;
```

Comment:
Styles applied to the composable elements should be scoped to layout concerns only, ping me if you have questions